.github/workflows/ci.yml

```diff
@@ -60,24 +60,32 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+        python-version: [
+          "2.7",
+          "3.5",
+          "3.6",
+          "3.7",
+          "3.8",
+          "3.9",
+          "3.10",
```

Comment:
I put these one-per-line to avoid a conflict with https://github.com/elastic/elasticsearch-dsl-py/pull/1606, something which might happen in the future too.